DESCRIPTION

```diff
@@ -1,18 +1,31 @@
 Package: fontquiver
 Title: Set of Installed Fonts
-Version: 0.1.0.9000
-Authors@R: person("Lionel", "Henry", , "lionel.hry@gmail.com", c("cre", "aut"))
+Version: 0.1.0.9001
+Authors@R: c(
+    person("Lionel", "Henry", , "lionel@rstudio.com", c("cre", "aut")),
+    person("RStudio", role = "cph"),
+    person("George Douros", role = "cph"))
 Description: Provides a set of fonts with permissive licences. This is
     useful when you want to avoid system fonts to make sure your
     outputs are reproducible.
 Depends:
     R (>= 3.3.0)
 Imports:
-    fontBitstreamVera (>= 0.1.0)
+    fontBitstreamVera (>= 0.1.0),
+    fontLiberation (>= 0.1.0)
 Suggests:
     testthat,
     htmltools
-License: GPL-3
+License: file LICENSE
```

Comment:
What is the meaning of `|` for the License field? I can't find this information in R exts